packages/scripts/src/upgrade-tests/voting2/0_Deploy.ts

```diff
@@ -101,27 +110,49 @@ async function main() {
   const proposerV2 = await proposerFactory.deploy(votingToken.address, defaultBond, governorV2.address, finder.address);
   console.log("Deployed ProposerV2: ", proposerV2.address);
 
-  console.log("6. Set ProposerV2 as the proposer of GovernorV2");
+  console.log("6. Deploying EmergencyProposer");
+  const quorum = hre.web3.utils.toWei("10000000", "ether");
+  // In localhost network we allow to not set the emergency executor address and default to the first account.
+  if (!process.env[EMERGENCY_EXECUTOR] && hre.network.name != "localhost") throw new Error("No emergency executor set");
+  const emergencyExecutor = process.env[EMERGENCY_EXECUTOR] || (await hre.ethers.getSigners())[0].address;
+  const multisig = (await hre.ethers.getSigners())[0].address;
+  const emergencyProposerFactory: EmergencyProposerEthers__factory = await getContractFactory("EmergencyProposer");
+  const emergencyProposer = await emergencyProposerFactory.deploy(
+    votingToken.address,
+    quorum,
+    governorV2.address,
+    multisig
```

Comment:
Shouldn't this be set to `emergencyExecutor` as provided in env (or default address when on localhost)?

Reply:
You are totally right, thanks for spotting! Fixed.